### PR TITLE
Activity log updated

### PR DESCRIPTION
**Pull request by rkzel**
Events are: opening of the issue, placing/updating associated bounty,
applying for work, review (and approval/rejection), submitting work
(and approval/rejection).

Changes: work/application Panels are opened by Links instead of buttons,
descriptions and comments are not shown, user login does not link
to GitHub, descriptions of events are as per design (slightly changed).

Before:

Screenshot from 2019-11-26 15-52-01

After:

Screenshot from 2019-11-26 18-02-10

### CODE REVIEW

File: apps/projects/app/components/Content/IssueDetail/EventsCard.js [hunk 1]
```diff
@@ -28,13 +25,6 @@ const IssueEvent = ({ user, ...props }) => {
         </IssueEventAvatar>
         <IssueEventDetails>
           <Text.Block size="small">
-            <Link
-              href={user.url}
-              target="_blank"
-              style={{ textDecoration: 'none', color: `${theme.link}` }}
-            >
-              {user.login}
```
Comment (chadoh) [inline apps/projects/app/components/Content/IssueDetail/EventsCard.js]: I want to double-check that we want to remove the link to the user's profile on GitHub. That seems useful to me.

Comment (stellarmagnet) [reply apps/projects/app/components/Content/IssueDetail/EventsCard.js]: The avatar should be clickable. Is that not the case?

Comment (rkzel) [reply apps/projects/app/components/Content/IssueDetail/EventsCard.js]: It wasn't the case before, it is now. One additional question then:

Screenshot from 2019-11-27 14-54-59

In panels it's the old way - avatar photo and username, usename being the link. Adjust? Username as plain text, avatart photo as link?

Comment (rkzel) [reply apps/projects/app/components/Content/IssueDetail/EventsCard.js]: I realized there is more to be fixed in the panels. Needs separate ticket and PR. Created #1696 for it.

**Pull request merged**